docs/spring-boot-runtime/master.adoc

```diff
@@ -14,6 +14,20 @@ include::topics/sb-tomcat-runtime-overview.adoc[leveloffset=+1]
 
 {SpringBoot} has a version of its runtime that is tested and verified to run on OpenShift. Some components, for example, the Embedded Tomcat Container and the Java container image, are provided as part of a Red Hat subscription when used with {SpringBoot}.
 
+The features below are available as a collection of missions and quickstarts link:{link-launcher-oso}[ on {OpenShiftOnline}] or your {OpenShiftLocal}.
+
+.Features and Frameworks Summary
+[format="csv", options="header"]
+[cols="3,10,3,6"]
+|===
+Feature,Problem Addressed, Native, Framework
+Circuit Breaker, Switch between services and continue to operate incoming requests without interruption,No, Spring Cloud Kubernetes - Hystrix
```

Comment:
Problem addressed: Switch between services and continue to process incoming requests without interruption in case of service failure.